.github/actions/build-e2e-matrix/action.yml

```diff
@@ -50,6 +50,7 @@ runs:
             ["visualizations-tabular", {} ],
             ["oss-subset", { edition: 'oss', context: "special" } ],
             ["slow", { runner: beefierRunner, context: "special" } ],
+            ["flaky", { runner: beefierRunner, context: "special" } ],
```

Comment:
Historically, running same tests on different runners yielded completely different results. I'm concerned that we'd maybe mask the underlying problem (or even potentially discover new ones specific to BuildJet runners).

IMO, we should solve the flakiness for the slowest and the least powerful runners, which are currently the default GitHub runners.